modules/host/host.go

```diff
@@ -66,13 +66,13 @@ var (
 type Host struct {
 	// RPC Tracking - atomic variables need to be placed at the top to preserve
 	// compatibility with 32bit systems.
-	atomicErroredCalls   uint64
-	atomicMalformedCalls uint64
-	atomicDownloadCalls  uint64
-	atomicRenewCalls     uint64
-	atomicReviseCalls    uint64
-	atomicSettingsCalls  uint64
-	atomicUploadCalls    uint64
+	atomicErroredCalls      uint64
+	atomicUnrecognizedCalls uint64
+	atomicDownloadCalls     uint64
+	atomicRenewCalls        uint64
+	atomicReviseCalls       uint64
+	atomicSettingsCalls     uint64
+	atomicUploadCalls       uint64
```

Comment:
maybe. But then you don't get to call the fields `atomic`. And your top level struct isn't flat, which I think makes for much cleaner code within a function.

`atomic.AddUint64(&h.atomicErroredCalls)`
`atomic.AddUint64(&h.atomicRPCTracking.ErroredCalls)`

I'd prefer the flat struct and some extra copying I think.